src/styles/Search.css

```diff
@@ -0,0 +1,27 @@
+.search-form {
+  display: flex;
+  flex-direction: row;
```

Comment:
Since `flex-direction: row` is the default behaviour you dont have to declare it